flixel/input/mouse/FlxMouse.hx

```diff
@@ -51,6 +51,11 @@ class FlxMouse extends FlxPointer implements IFlxInputManager
 	 * sits at the right "height". Not used on flash with the native cursor API.
 	 */
 	public var cursorContainer(default, null):Sprite;
+	
+	/**
+	 * This is just a reference to the current cursor image, if there is one.
+	 */
+	public var cursor:Bitmap;
```

Comment:
lets make it read-only, you'll still be able to set properties. Also should mark it as nullable which is something I want to start doing
```suggestion
	public var cursor(default, null):Null<Bitmap>;
```